exec/exec.go

```diff
@@ -79,20 +80,23 @@ type VMContext interface {
 
 	CreateNewActor(addr address.Address, code cid.Cid, initalizationParams interface{}) error
 
-	// TODO: Remove these when Storage above is completely implemented
-	ReadStorage() ([]byte, error)
-	WriteStorage(interface{}) error
+	Verifier() Verifier
 }
 
 // Storage defines the storage module exposed to actors.
 type Storage interface {
-	// TODO: Forgot that Put() can fail in the spec, need to update.
 	Put(interface{}) (cid.Cid, error)
 	Get(cid.Cid) ([]byte, error)
 	Commit(cid.Cid, cid.Cid) error
 	Head() cid.Cid
 }
 
+// Verifier is the proof verification interface
+type Verifier interface {
+	VerifySeal(req verification.VerifySealRequest) (verification.VerifySealResponse, error)
+	VerifyPoSt(req verification.VerifyPoStRequest) (verification.VerifyPoStResponse, error)
```

Comment:
Since we're abstracting it, I'd strongly consider removing the Request/Response structs in favour of simple parameters. It's not an RPC at this level.

Reply:
Oh, I see that this interface is intended to be satisfied by the existing RustVerifier. My comment probably applies at that level, but no need to do anything about it right now. Just FYI.